### ISSUE TITLE
add option to set different random seed for each process

### ISSUE THREAD

**Issue by mmckerns** [uqfoundation/pathos#40]
`multiprocessing` has a known issue where certain global random states (and thus the seeds) are copied to all the spawned processes (most notably for anything depending on `numpy.random`).  A potential fix for this is to provide `processing` (the `pathos` fork) or `pathos.multiprocessing` with a ["set random_state" function](https://github.com/uqfoundation/mystic/blob/fe08977e0baa25d13c32bd1bd06efc32ab943be6/mystic/tools.py#L129).  Then optionally, provide some API extension to enable easy triggering of the "special" seeding (i.e. generating a new random state for each process).

**Comment by rubendibattista** [comment]
Has this been implemented at the end?

**Comment by mmckerns** [comment]
I don't think so.  The code (linked above) is obviously there and working... but it's currently part of `mystic`.  It probably should be moved to `multiprocess` or `pathos.multiprocessing`.
